.github/PULL_REQUEST_TEMPLATE/standard-change.md

```diff
@@ -4,7 +4,7 @@
 
 ## Related links
 
-<!-- Write the links related to this PR. -->
+<!-- Write the links related to this PR. If there are private links, write they are, e.g., TIERIV INTERNAL LINK. -->
```

Comment:
Like this?

https://developers.google.com/style/word-list
![image](https://user-images.githubusercontent.com/31987104/191217065-2f8d4f15-0e30-4986-8eba-e16b24fd651b.png)

```suggestion
<!-- Write the links related to this PR. Private links should be clearly marked as private, for example, 'TIER IV INTERNAL LINK'. -->
```

Reply:
Thank you!